[이아영] 6주차 과제 제출

otwaylee opened the pull request:
## 결과 소개

[☘️6주차 README 바로가기](https://github.com/otwaylee/efub5-frontend-assignment-2/tree/main/week06)

https://github.com/user-attachments/assets/07cf063c-e101-48c3-a257-0a66f6778574

otwaylee committed:
week04 실습 준비

otwaylee committed:
init: 초기 설정

otwaylee committed:
feat: 1부 실습

otwaylee committed:
chore: tailwind, eslint-plugin-simple-import-sort

otwaylee committed:
chore: taildwind 자동 정렬 설정

otwaylee committed:
chore: eslint prettier 충돌 방지 설정

otwaylee committed:
feat: 헤더, 액션바 구현

otwaylee committed:
feat: 게시글 리스트

otwaylee committed:
feat: 수정, 삭제

otwaylee committed:
design: 스타일 수정

otwaylee committed:
feat: 임시 회원가입 기능

otwaylee committed:
docs: readme 추가

otwaylee committed:
Update README.md

seooyuun commented on week06/my-app/src/app/signup/page.tsx:
```diff
@@ -0,0 +1,52 @@
+  async function handleSubmit(e: React.FormEvent) {
+    e.preventDefault();
+
+    const res = await fetch("/api/signup", {
```
`try-catch`를 적용하면 네트워크 예외에도 안전한 코드가 될 것 같습니다!

seooyuun reviewed (commented):
아영님! 과제 하느라 수고 많으셨습니다!!
전체적으로 UI가 깔끔해서 매우 보기 좋았던 것 같습니다! 회원가입 폼의 기본 구조가 잘 짜여있고 상태 관리 흐름도 깔끔합니다! 고생하셨어요👊🏻

otwaylee committed:
continue week06

otwaylee committed:
feat: 실습 1부

otwaylee committed:
feat: 실습 2부

otwaylee committed:
feat: 좋아요 기능 구현
- 로그인 한 유저만 좋아요 가능
- 중복으로 좋아요 방지
- 좋아요 수 확인 가능

otwaylee committed:
feat: 삭제 기능 구현

otwaylee committed:
feat: 댓글 기능 구현

otwaylee committed:
feat: 관리자 권한 기능 구현
- 사용자 스키마 role 필드 추가 (기본값 normal)
- 로그인 페이지에 로그인 방법 선택
- admin 권한  게시글 수정 삭제 모두 가능

otwaylee committed:
docs: readme

otwaylee committed:
Update README.md
